nornir/plugins/functions/text/__init__.py

```diff
@@ -115,7 +161,8 @@ def print_result(
     vars: List[str] = None,
     failed: bool = False,
     severity_level: int = logging.INFO,
-) -> None:
+    return_output: bool = False,
+) -> Union[None, str]:
```

Comment:
with my proposal of having the user pass the `StringIO` this doesn't need to change.